src/app/account-app/account-app.component.ts

```diff
@@ -32,17 +33,27 @@ export class AccountAppComponent {
     @ViewChild(MatSidenav) sideMenu: MatSidenav;
     sideMenuOpened = true;
     rmm6tooltip = 'This area isn\'t upgraded to Runbox 7 yet and will open in a new tab';
+    isMainAccount: boolean;
 
     constructor(
-        public cart:        CartService,
-        public mobileQuery: MobileQueryService,
-               router:      Router,
+        public  cart:        CartService,
+        public  mobileQuery: MobileQueryService,
+                rmmapi:      RunboxWebmailAPI,
+                router:      Router,
     ) {
         this.sideMenuOpened = !mobileQuery.matches;
         this.mobileQuery.changed.subscribe(mobile => {
             this.sideMenuOpened = !mobile;
         });
 
+        rmmapi.me.subscribe((me: RunboxMe) => {
+            if (me.owner) {
```

Comment:
Minor: could be shortened to `this.isMainAccount = !me.owner`

Reply:
Did the thing